test/remote/gateways/remote_decidir_plus_test.rb

```diff
@@ -186,6 +189,16 @@ def test_successful_purchase_with_fraud_detection
     assert_equal({ 'status' => nil }, response.params['fraud_detection'])
   end
 
+  def test_successful_purchase_with_card_brand
+    options = @options.merge(card_brand: 'cabal')
+
+    assert response = @gateway_purchase.store(@cabal)
+    payment_reference = response.authorization
+
+    response = @gateway_purchase.purchase(@amount, payment_reference, options)
+    assert_success response
```

Comment:
Nit: do you think it would be worth including an assertion here showing that `response.params['payment_method_id']` is the expected value?

Reply:
Good suggestion, added.